lib/internal/modules/run_main.js

```diff
@@ -54,6 +66,10 @@ function runMainESM(mainPath) {
   }));
 }
 
+/**
+ * Handle process exit events around the main entry point promise.
+ * @param {Promise} promise Main entry point promise
```

Comment:
```suggestion
 * @param {Promise<unknown>} promise Main entry point promise
```

Reply:
This isn’t really any better, in my opinion. I’m not sure _how_ to type what this promise is, but it’s not unknown, it’s just something that we haven’t defined the type for yet.

The goal in this PR isn’t to be comprehensive and type everything possible, just to add minimal JSDoc coverage that satisfies the lint rule, for now. The JSDoc annotations can always be improved in follow-ups.